System/System.cpp

```diff
@@ -20,9 +22,22 @@ namespace RTE {
 	const std::string System::s_ZippedModulePackageExtension = ".rte.zip";
 	const std::unordered_set<std::string> System::s_SupportedExtensions = { ".ini", ".txt", ".lua", ".cfg", ".bmp", ".png", ".jpg", ".jpeg", ".wav", ".ogg", ".mp3", ".flac" };
 
-/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+#ifdef __unix__
+	const std::filesystem::path System::s_BaseDataDirectory{BASEDATAPATH};
+	std::filesystem::path System::s_TempDirectory{};
+#endif
+
+	/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 	void System::Initialize() {
+#if defined(__unix__) && !LINUX_PORTABLE
```

Comment:
Can't because LINUX_PORTABLE is defined explicitly as 1 or 0

Reply:
Not sure what you misread but you're actually right about this.
Looks like checking `ifdef/idndef/defined/!defined` only cares about the macro actually being defined, no matter the value.
So it would only work if you were going from no macro to yes macro rather than macro = 0 to macro = 1.
So yeah you can ignore and resolve, or do whatever else method you found in the docs.
You could maybe do `#if (MACRO == 1)` etc so it's immediately obvious what is going on.